front/components/GoogleDriveFoldersPickerModal.tsx

```diff
@@ -137,6 +137,13 @@ export default function GoogleDriveFoldersPickerModal(props: {
                         />
                       </div>
                     )}
+                    {!isLoading && folders.length === 0 && (
+                      <div className="mt-2 flex h-[500px] items-center justify-center overflow-y-auto rounded border-2 border-x-gray-100 text-gray-500">
+                        <div>
+                          Only shared drives are supported at the moment.
```

Comment:
NIT suggestion: "No folder available for synchronization (only shared drives are supported at the moment)".

Reply:
Thanks! Updated :+1: